walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -89,8 +131,22 @@
     }
     // If at the last event and there is still time in the day.
     if (nextTime < END_OF_DAY) {
-      query.add(TimeRange.fromStartEnd(nextTime, END_OF_DAY, /* inclusive= */ true));
+      query.add(fromStartEnd(nextTime, END_OF_DAY, /* inclusive= */ true));
     }
     return query;
   }
+
+  public Collection<String> combineMandatoryAndOptional(
+      Collection<String> mandatoryAttendees, Collection<String> optionalAttendees) {
+    // Collection of Strings representing all attendees ( Mandatory and Optional )
+    Collection<String> allAttendees = new ArrayList<>();
+    allAttendees.addAll(optionalAttendees);
+    allAttendees.addAll(mandatoryAttendees);
+    return allAttendees;
+  }
+
+  public List<com.google.sps.TimeRange> populateTimeRanges(
+      Collection<com.google.sps.Event> events, Collection<String> attendees, int desiredDuration) {
+    return (List<com.google.sps.TimeRange>) getTimeRanges(events, attendees, desiredDuration);
```

Comment:
does this need to be cast?

Reply:
Yes  it was as the function returned a collection, but i changed it to return a list